pkg/sdk/testint/security_integrations_gen_integration_test.go

```diff
@@ -0,0 +1,380 @@
+		details, err := client.SecurityIntegrations.Describe(ctx, id)
+		require.NoError(t, err)
+
+		assertSCIMDescribe(details, "true", "", "GENERIC_SCIM_PROVISIONER", "false", "altered")
+
+		unsetRequest := sdk.NewAlterSCIMIntegrationSecurityIntegrationRequest(id).
+			WithUnset(
+				sdk.NewSCIMIntegrationUnsetRequest().
+					WithNetworkPolicy(sdk.Bool(true)).
+					WithSyncPassword(sdk.Bool(true)),
+			)
+		err = client.SecurityIntegrations.AlterSCIMIntegration(ctx, unsetRequest)
+		require.NoError(t, err)
+
+		details, err = client.SecurityIntegrations.Describe(ctx, id)
+		require.NoError(t, err)
+
+		assertSCIMDescribe(details, "true", "", "GENERIC_SCIM_PROVISIONER", "true", "altered")
+	})
+
+	t.Run("Alter - set and unset tags", func(t *testing.T) {
+		tag, tagCleanup := testClientHelper().Tag.CreateTag(t)
+		t.Cleanup(tagCleanup)
+
+		id := testClientHelper().Ids.RandomAccountObjectIdentifier()
+		createSCIMIntegration(t, id, nil)
+
+		tagValue := "abc"
+		tags := []sdk.TagAssociation{
+			{
+				Name:  tag.ID(),
+				Value: tagValue,
+			},
+		}
+		alterRequestSetTags := sdk.NewAlterSCIMIntegrationSecurityIntegrationRequest(id).WithSetTags(tags)
+
+		err := client.SecurityIntegrations.AlterSCIMIntegration(ctx, alterRequestSetTags)
+		require.NoError(t, err)
+
+		returnedTagValue, err := client.SystemFunctions.GetTag(ctx, tag.ID(), id, sdk.ObjectTypeIntegration)
+		require.NoError(t, err)
+
+		assert.Equal(t, tagValue, returnedTagValue)
+
+		unsetTags := []sdk.ObjectIdentifier{
+			tag.ID(),
+		}
+		alterRequestUnsetTags := sdk.NewAlterSCIMIntegrationSecurityIntegrationRequest(id).WithUnsetTags(unsetTags)
+
+		err = client.SecurityIntegrations.AlterSCIMIntegration(ctx, alterRequestUnsetTags)
+		require.NoError(t, err)
+
+		_, err = client.SystemFunctions.GetTag(ctx, tag.ID(), id, sdk.ObjectTypeIntegration)
+		require.Error(t, err)
+	})
+
+	t.Run("Drop", func(t *testing.T) {
+		id := testClientHelper().Ids.RandomAccountObjectIdentifier()
+		createSCIMIntegration(t, id, nil)
+
+		si, err := client.SecurityIntegrations.ShowByID(ctx, id)
+		require.NotNil(t, si)
+		require.NoError(t, err)
+
+		err = client.SecurityIntegrations.Drop(ctx, sdk.NewDropSecurityIntegrationRequest(id))
+		require.NoError(t, err)
+
+		si, err = client.SecurityIntegrations.ShowByID(ctx, id)
+		require.Nil(t, si)
+		require.Error(t, err)
+	})
+
+	t.Run("Drop non-existing", func(t *testing.T) {
+		id := sdk.NewAccountObjectIdentifier("does_not_exist")
+
+		err := client.SecurityIntegrations.Drop(ctx, sdk.NewDropSecurityIntegrationRequest(id))
+		assert.ErrorIs(t, err, sdk.ErrObjectNotExistOrAuthorized)
+	})
+
+	t.Run("Describe", func(t *testing.T) {
+		id := testClientHelper().Ids.RandomAccountObjectIdentifier()
+		createSCIMIntegration(t, id, nil)
+
+		details, err := client.SecurityIntegrations.Describe(ctx, id)
+		require.NoError(t, err)
+
+		assertSCIMDescribe(details, "false", "", "GENERIC_SCIM_PROVISIONER", "true", "")
+	})
+
+	t.Run("ShowByID", func(t *testing.T) {
```

Comment:
let's add a test for a proper show by id (check in other tests)

Reply:
Done.